ocs_ci/deployment/provider_client/storage_client_deployment.py

```diff
@@ -173,12 +173,13 @@ def provider_and_native_client_installation(
         self.odf_subscription_on_provider()
 
         # Check for rook ceph pods
-        assert self.pod_obj.wait_for_resource(
-            condition="Running",
-            selector="app=rook-ceph-operator",
-            resource_count=1,
-            timeout=600,
-        )
+        if self.ocs_version < version.VERSION_4_19:
+            assert self.pod_obj.wait_for_resource(
+                condition="Running",
+                selector="app=rook-ceph-operator",
+                resource_count=1,
```

Comment:
Maybe stupid question: Why are we checking for this only for versions less than 4.19?
Wont we also have it in 4.19 and above?

Reply:
ack and +1 to Elena

Reply:
Based on discussion with Daniel, this will be handled in follow-up PR.